environments/dev-env/build-edge-syncer.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
(1) Naming this script "build-<something>" is confusing, because what it does is not usually called "building".

(2) We already have `bin/mailbox-prep.sh` that gets the same job done.  I do not think we want to two scripts.  If the existing script lacks needed features (such as a `--verbose` flag) then let's fix that rather than make a substantially duplicate script.